Automattic/wc-calypso-bridge
# Remove Jetpack Connection step in tax task on a free trial

### Changes proposed in this Pull Request:



Closes #1146 

This PR removes Jetpack connection step in WooCommerce Tax setup step as atomic sites are already connected to Jetpack. 








### How to test the changes in this Pull Request:

1. Checkout this branch in your setup and make sure Jetpack is connected.
2. Click `Add tax rates` task in WooCommerce Home
3. Click `WooCommerce Tax`
4. Confirm the changes

![Screen Shot 2023-07-30 at 2 05 56 PM](https://github.com/Automattic/wc-calypso-bridge/assets/4723145/8ed22285-d0fa-48e0-ac7c-3658a2c2a0e7)
![Screen Shot 2023-07-30 at 2 05 45 PM](https://github.com/Automattic/wc-calypso-bridge/assets/4723145/b9ad9adc-ce72-4588-9ddf-b684c31cd463)




### Other information:




### FOR PR REVIEWER ONLY:
